src/shared/views/RecommendedCards.js

```diff
@@ -77,13 +78,119 @@ type OrderedProject = {
 
 type Props = {
     announcement: Object,
+    firebase: Object,
     navigation: NavigationProp,
     projects: Array<OrderedProject>,
+    teamId: ?string,
+};
+
+// request only active projects from firebase (status === 'active')
+// firebase doesn't allow multiple query params, so for project types we filter in render()
+// but here we can still limit to 20 projects maximum
+// `path` defines where the resulting data is copied in the redux store
+// (state.firebase.ordered.projects in this case, because we've asked for `orderByChild`)
```

Comment:
as a general comment we could add here that this query needs to be allowed by the firebase db rules set in the back end

Reply:
Added.